woocommerce/payment-gateway/Blocks/Gateway_Checkout_Block_Integration.php

```diff
@@ -114,8 +114,7 @@ public function get_payment_method_script_handles() {
 	 *
 	 * @return array<string, mixed>
 	 */
-	public function get_script_data()
-	{
+	public function get_payment_method_data() : array {
```

Comment:
@unfulvio-godaddy this method seems to be required for payment gateways so their settings are available via `getSetting` in frontend.